test/unittest/blas2/blas2_spr_test.cpp

```diff
@@ -26,18 +26,17 @@
 #include "blas_test.hpp"
 
 template <typename scalar_t>
-using combination_t =
-    std::tuple<std::string, char, char, index_t, scalar_t, index_t>;
+using combination_t = std::tuple<std::string, char, index_t, scalar_t, index_t>;
 
 template <typename scalar_t, helper::AllocType mem_alloc>
 void run_test(const combination_t<scalar_t> combi) {
   std::string alloc;
   index_t n;
   index_t lda_mul;
   index_t incX;
-  char layout, uplo;
```

Comment:
I think we can keep the `layout` option here, since the code practically works for both `row major` and `col major` layouts. The only reason why this option was kept here and not introduced in the library was because we don't have a dedicate support for `row major` layout yet.